### PR TITLE
WorldGenereratorStuff

### PR DESCRIPTION
**Pull request by Duckos-Mods**

### CODE REVIEW

File: AmethystRuntime/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Amethyst Options
-set(MOD_VERSION "1.3.1")
+set(MOD_VERSION "1.4.0")
```
Comment (FrederoxDev) [inline AmethystRuntime/CMakeLists.txt:2]: I swear u already made this change lol

**Review by FrederoxDev — APPROVED** [review]

**Comment by FrederoxDev** [comment]
u got conflicts

File: AmethystRuntime/CMakeLists.txt (hunk 1 above)
Comment (Duckos-Mods) [reply AmethystRuntime/CMakeLists.txt]: I thought i did too

**Comment by FrederoxDev** [comment]
Thanks!

**Pull request merged**